src/App.tsx

```diff
@@ -1,24 +1,136 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import {
+  getTodos, updateTodos,
+} from './api/todos';
+import { Filter } from './types/Filter';
+import { TodoList } from './components/TodoList';
+import { USER_ID } from './api/userId';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Error } from './components/Error';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
it would be great introduce type for errors: `enum ErrorType` and use it everywhere